tests/config/pytest_modules.yml

```diff
@@ -353,6 +353,10 @@ bedtools/multiinter:
   - modules/nf-core/bedtools/multiinter/**
   - tests/modules/nf-core/bedtools/multiinter/**
 
+bedtools/slop:
```

Comment:
```suggestion
bedtools/shift:
```

Reply:
sorry...!

Reply:
No problem! (you wouldn't want to know how many times they had to fix copy paste errors from me 😁)